.env

```diff
@@ -0,0 +1,2 @@
+DB_CONNECT=testing
```

Comment:
does your local version have a .env in the .gitignore?

Reply:
was going to ask this also, may need to add the .env to your .gitignore

Reply:
MogoURI has the server pw so will need to be in .gitignore

Reply:
Not sure why it pushed the .env this go around, will fix and re push soon